airflow/provider_info.schema.json

```diff
@@ -71,13 +71,20 @@
           "type": "string"
       }
     },
-      "notifications": {
+    "notifications": {
           "type": "array",
           "description": "Notification class names",
           "items": {
               "type": "string"
           }
-      },
+    },
+    "executors": {
```

Comment:
Niiice, I know I'm late to this PR but this is great! I was planning on adding a section to the schema exactly like this when we're ready to land our first AWS executor. Thanks for beating us to it @potiuk :smile: